src/test/scala/is/hail/methods/AggregatorSuite.scala

```diff
@@ -123,6 +126,31 @@ class AggregatorSuite extends SparkSuite {
     p.check()
   }
 
+  @Test def testMaxMin() {
+    val rdd1 = sc.parallelize(Seq(
+      Annotation("a",  0, null, -1, -1.0, null),
+      Annotation("a", -1, null, -2, 1.0, null)), numSlices = 1)
+
+    val rdd2 = sc.parallelize(Seq(
+      Annotation("a",  1, -1, -3, null, null),
+      Annotation("a",  0, -2, -4, 0.0, null)), numSlices = 1)
+
+    val rdd = rdd1.union(rdd2)
```

Comment:
I guess I don't know what you mean by "the null behavior" and how this achieves that.

I guess you'd want to test a combOp of a non-null and a null, but you don't do that.  Or a seqOp of a non-null followed by a null, but you don't do that either.

Yes, I think the merge is less clear than than sc.parallelize(..., 2).